charts/trino/values.yaml

```diff
@@ -601,6 +601,16 @@ coordinator:
   #    protocol: TCP
   # ```
 
+  additionalConfigProperties: []
+  # additionalConfigProperties -- [Additional config
+  # properties applied to coordinator only](https://trino.io/docs/current/admin/properties.html).
+  # @raw
+  # Example:
+  # ```yaml
+  #  - web-ui.preview.enabled=true
```

Comment:
Can you add these to https://github.com/trinodb/charts/blob/main/tests/trino/test-values.yaml so we know these are correct examples?